### ISSUE TITLE
Use WSS instead of WS

### ISSUE THREAD

**Issue by norbertdragan** [chips-blockchain/pangea-poker#33]
HTTPS is also more secure, and in order to go to production on the web, we would have to switch.

I would like to use netlify for continous deployment, but currently we are using `ws` which only work on non-secure `http` connections, and is not supported by Netlify. 

In order to use `https`, we need to use `wss`.

**Comment by norbertdragan** [comment]
Any thoughts on this @sg777?

**Comment by sg777** [comment]
Agreed, going forward we must use wss. In the backend im using libwebsockets library, I'll check in order to use wss any certificate generation should be needed in the backend and will check for any dependencies.

**Comment by norbertdragan** [comment]
@sg777 have you had any chance to look at it? What changes do we need to do in order to get WSS working?

**Comment by norbertdragan** [comment]
WSS is not mandatory with Electron, so I reduced the priority of this issue to `P2`.
